scadalts-ui/src/views/AlarmTabs.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+	<div class="alarms">
+		<v-card>
+			<v-tabs
+			v-model="tab"
+			@change="changeTab"
+			>
+				<v-tab
+					v-for="item in items"
+					:key="item.tab"
+					:class="{'blink':(item.tab === 'plc' && newAlarms)}"
+				>
+					{{ item.content }}
+				</v-tab>
+			</v-tabs>
+		</v-card>				
+        <router-view></router-view>
+
+	</div>
+</template>
+
+<script>
+import Components from '@min-gb/vuejs-components';
+
+export default {
+	name: 'alarmTabs',
+	components: {
+		...Components,
+	},
+	mounted() {
+		this.tab = 0
+		this.changeTab(this.tab)
+		const interval = setInterval(() => {
+			this.getAlarms();
+		}, 1000);
+	},
+	data () {
+      return {
+		newAlarms: 0,
+		tab: 0,
+		items: [
+			{ tab: 'scada', content: 'Scada' },
+			{ tab: 'plc', content: 'PLC' },
+			{ tab: 'plc-history', content: 'PLC History' },
+		],
+      }
+    },
+	methods: {
+		getAlarms() {
+			this.$store.dispatch('getLiveAlarms', { offset: 0, limit: 1 })
+			.then((ret) => {
+				this.newAlarms = ret.length
+			});
+		},
+		changeTab(index) {
+			this.$router.push({ path: `/alarms/${this.items[index].tab}` });
+		},
+	},
+};
```

Comment:
Just after "methods" property add a new one called "watch" property to add the feature that will follow the router change and change the active tab index depending on the routing.

```
watch: {
		$route(to) {
			switch (to.name) {
				case 'scada':
					this.tab = 0
					break
				case 'plc':
					this.tab = 1
					break
				case 'plc-history':
					this.tab = 2
					break
			}
		}
	}
```
I share with You an example code that works and now even using the "flag" the tab is updated.

Reply:
done